source/MapSalesPanel.cpp

```diff
@@ -220,12 +221,40 @@ void MapSalesPanel::DrawKey() const
 		RingShader::Draw(pos, OUTER, INNER, MapColor(VALUE[i]));
 		font.Draw(KeyLabel(i), pos + textOff, isSelected ? bright : dim);
 		if(onlyShowSoldHere && i == 2)
-		{
 			// If we're filtering out items not sold here, draw a pointer.
 			PointerShader::Draw(pos + Point(-7., 0.), Point(1., 0.), 10.f, 10.f, 0.f, bright);
-		}
 		pos.Y() += 20.;
 	}
+	
+	if(MapPanel::maxColor > 1. || MapPanel::minColor < 1.)
+	{
+		const Color &medium = *GameData::Colors().Get("medium");
+		const Sprite *sales = SpriteSet::Get("ui/sales");
+		
+		int height = max(SelectedInfo().AttributesHeight(), CompareInfo().AttributesHeight());
+		pos = Screen::TopRight() + Point(-110., height + sales->Height() / 2.);
```

Comment:
```suggestion
		pos = Screen::TopRight() + Point(-110., height + sales->Height() / 2. + 15);
```

The UI images overlap a bit here, so they can use a little padding.